### ISSUE TITLE
Haptic feedback should be independent on system settings

### ISSUE THREAD

**Issue by MichalLebeda** [8VIM/8VIM#184]
I think that haptic feedback should not require having global system haptic feedback. I am not a fan of global haptic feedback as there is no reason for it IMO. This keyboard is a special case which is much better with haptic feedback. I would suggest (and volunteer) to implement independent haptic feedback with separate preference for vibration strength for these actions:
1. crossing diagonal separator (would be super helpful with smaller circles)
2. writing a letter

**Comment by flide** [comment]
That would be great.

To be honest I didn't even knew we can do something like that in the first
place...

**Comment by analpi** [comment]
Another FOSS keyboard I'm using, [FlorisBoard](https://github.com/florisboard/florisboard),  has an independent haptic feedback option that may or may not be helpful to look at. (I'm not a coder, sadly.)

**Comment by mentalisttraceur** [comment]
Agreed, but I would like to mention that some users might want an option to have the settings linked.
